tox.ini

```
@@ -86,6 +86,7 @@ envlist =
     mako_contrib-{py27,py34,py35,py36,py37}-mako{010,100}
     molten_contrib-py{36,37}-molten{070,072}
     mongoengine_contrib-{py27,py34,py35,py36,py37}-mongoengine{015,016,017,018,latest}-pymongo{latest}
+    multiprocessing_contrib-py{27,36,37}
```

```suggestion
    multiprocessing_contrib-py{27,34,35,36,37}
```

We do still officially support them right now :)